### PR TITLE
feat: terms and conditions

### PR DESCRIPTION
**Pull request by rbadici-plenty**
## Why:

Closes: #ID

## Checklist before requesting a review
- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/pages/Unsubscribe.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <NewsletterUnsubscribeForm />
+    </div>
+  </div>
+</template>
+
+<script>
+import { ref } from '@nuxtjs/composition-api';
+import NewsletterUnsubscribeForm from '~/components/MyAccount/NewsletterUnsubscribeForm.vue';
+
+export default {
```
Comment (maxiroellplenty) [inline apps/web/pages/Unsubscribe.vue:16]: also this has to be script setup from compositon api not options api

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: apps/web/pages/CancellationForm.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+});
+
+const getHTMLTexts = () => {
+  return data.value.htmlText ?? '';
+};
+</script>
+<style>
+.legal-texts {
+  padding: 20px;
+}
```
Comment (FabianGerke) [inline apps/web/pages/CancellationForm.vue:19]: Remove css and use the class p-5 instead.
https://tailwindcss.com/docs/padding

File: apps/web/pages/CancellationForm.vue [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <div class="legal-texts" v-html="getHTMLTexts()" />
```
Comment (FabianGerke) [inline apps/web/pages/CancellationForm.vue:2]: replace legal-texts with p-5

**Review by FabianGerke — CHANGES_REQUESTED** [review]
Remove css and use tailwind in all your components.

File: apps/web/composables/useLegalInformation/useLegalInformation.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  const state = useState<UseLegalInformationState>('useLegalInformation', () => ({
+    data: {} as LegalInformationResponse,
+    loading: false,
+  }));
+
+  /**
+   * @description Function for fetching the category tree.
+   * @example
+   * getCategoryTree();
+   */
```
Comment (ksted) [inline apps/web/composables/useLegalInformation/useLegalInformation.ts:25]:
```suggestion
  /**
   * @description Function for legal information of the given type.
   * @param LegalTextsParams Type of the legal information
   * @example
   * getLegalTexts({type: 'CancellationForm',});
   * @example
   * getLegalTexts({type: 'CancellationRights',});
   * @example
   * getLegalTexts({type: 'LegalDisclosure',});
   * @example
   * getLegalTexts({type: 'PrivacyPolicy',});
   * @example
   * getLegalTexts({type: 'TermsConditions',});
   */
```

File: apps/web/lang/en.json [hunk 5]
```diff
@@ -192,6 +192,16 @@
...
+    },
+    "legal": {
+      "label": "Legal",
+      "subcategories": {
+        "termsAndConditions": "Terms and Conditions",
+        "cancellationRights": "Cancellation Rights",
+        "legalDisclosure": "Legal Disclosure",
+        "privacyPolicy": "Privacy Policy",
+        "cancellationForm": "Cancellation Form"
+      }
```
Comment (ksted) [inline apps/web/lang/en.json:204]:
```suggestion
    "legal": {
      "label": "Legal",
      "subcategories": {
        "cancellationForm": "Cancellation form",
        "cancellationRights": "Cancellation rights",
        "legalDisclosure": "Legal disclosure",
        "privacyPolicy": "Privacy policy",
        "termsAndConditions": "Terms and conditions",
      }
```
**todo**: Also add translations to `de.json`.
```
    "legal": {
      "label": "Rechtliches",
      "subcategories": {
        "cancellationForm": "Widerrufsformular",
        "cancellationRights": "Widerrufsbelehrung",
        "legalDisclosure": "Impressum",
        "privacyPolicy": "Datenschutzerklärung",
        "termsAndConditions": "AGB"
      }
```

**Review by ksted — CHANGES_REQUESTED** [review]

**Comment by ksted** [comment]
**todo**: Please also add the links to the terms and conditions and to the privacy policy in the checkout.
Screenshot 2023-08-24 at 07 59 49

**Review by ksted — APPROVED** [review]

**Review by FabianGerke — APPROVED** [review]

**Review by ksted — APPROVED** [review]

**Pull request merged**